generator/src/main/java/com/scottlogic/deg/generator/walker/ReductiveDecisionTreeWalker.java

```diff
@@ -38,7 +40,7 @@ public ReductiveDecisionTreeWalker(
     }
 
     /* initialise the walker with a set (ReductiveState) of unfixed fields */
-    public Stream<RowSpec> walk(DecisionTree tree) {
+    public Stream<RowSpec> walk(DecisionTree tree, GenerationConfig config) {
```

Comment:
This config should be injected in

Reply:
The `config` is not a dependency, therefore it shouldn't be injected as one.

Reply:
parameter has been removed as part of a refactor